### DISCUSSION TITLE
关于主题授权的建议

### DISCUSSION THREAD

**Discussion by xiaozhangStudent** [halo-dev/halo#5178] [Ideas]
### 你当前使用的版本

2.11.3

### 描述一下此特性

1·希望在下个版本添加域名授权系统
2.希望下个版本能够提供一次性代金券，或一次性百分比优惠券

**Comment by JohnNiang** [comment]
Hi @xiaozhangStudent ，建议提供一下期望添加主题授权以及代金券（优惠券）功能的原因是什么？提供更多的上下文，社区才能够正确理解你的需求。

/triage needs-information

**Comment by xiaozhangStudent** [comment]
开发者您好，目前的付费主题模式容易被泛滥使用，只要有一个人购买，其他人使用购买者的账号就可以使用，优惠券的作用是可以将主题以其他方式进行售卖

**Comment by JohnNiang** [comment]
Hi @xiaozhangStudent ，目前我们正在做的尝试：
- 为付费主题和插件提供 License 机制。不过因为主题是纯模板文件，很容易被复刻，实现起来不是那么容易。
- 应用市场已提供折扣功能，但开发者中心还未完成，所以主题/插件作者无法设置。

最后，我将会把当前 Issue 转换为 Discussion。

**Reply by ruibaby** [reply]
期待。
